tests/FunctionAllTest.php

```diff
@@ -9,9 +9,9 @@ class FunctionAllTest extends TestCase
     /** @test */
     public function shouldResolveEmptyInput(): void
     {
-        $mock = $this->createCallableMock();
-        $mock
-            ->expects(self::once())
+        /** @var callable $mock */
+        $mock = $_ = $this->createCallableMock();
+        $_->expects(self::once())
```

Comment:
Is this still needed? (see also below)

See alternative solution in https://github.com/reactphp/promise/pull/246/files#diff-abf1fa77a19f720052b75bc802df0260157b9064d8c3c12a80794f01e9acc065R49.

Reply:
Doesn't seem needed anymore, so no need for the alternative solution (currently).